device-telemetry/app/com/microsoft/azure/iotsolutions/devicetelemetry/services/runtime/StorageConfig.java

```diff
@@ -11,42 +11,31 @@
 
     private static final Logger.ALogger log = Logger.of(StorageConfig.class);
 
-    private final String storageType;
     private final String documentDbConnString;
     private final String documentDbDatabase;
     private final String documentDbCollection;
 
     public StorageConfig(
-        String storageType,
         String documentDbConnString,
         String documentDbDatabase,
         String documentDbCollection) {
 
-        this.storageType = storageType;
-        if (!this.storageType.equalsIgnoreCase("documentdb")) {
-            log.error("Unknown storage type: '{}'", this.storageType);
-        }
-
         this.documentDbConnString = documentDbConnString;
```

Comment:
since you're already renaming from documentdb->cosmosdb can you also rename the variables and method names with documentdb here?

Reply:
make sense!